e2e/nomostest/git-server.go

```diff
@@ -32,7 +33,9 @@ import (
 
 const testGitNamespace = "config-management-system-test"
 const testGitServer = "test-git-server"
-const testGitServerImage = testing.TestInfraArtifactRegistry + "/git-server:v1.0.0"
+
+// TODO: update the image tag with commit SHA of https://github.com/GoogleContainerTools/kpt-config-sync/pull/1145.
+const testGitServerImage = testing.TestInfraArtifactRegistry + "/git-server:v1.0.0-patch"
```

Comment:
gentle reminder: this can be updated now

Reply:
updated